wiki/Tournaments/OWC/2014/de.md

```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
-   2. Raumeinstellungen sind osu!, Team-Vs., Win Condition: 'Score'. Raumname muss "OWC 2014: TeamBlue vs TeamRed" lauten.
-   3. Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
-2. Den Spielern steht es zu 2 Beatmaps für die "Warm-Up" Phase zu spielen. Beatmaps mit fragwürdigem Inhalt sind verboten.
-3. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
-   1. Beide Vetos müssen benutzt werden.
-   2. Beide Kapitäne müssen `!roll` benutzen, um zu ermitteln, wer mit dem Verbannen der beiden Beatmaps beginnen soll. Der andere Kapitän darf nicht dieselben Beatmaps verbannen und muss daher andere Beatmaps auswählen, um Überlappungen zu vermeiden.
-4. Die Beatmaps werden abwechselnd von den beiden Kapitänen ausgewählt. Zuvor muss jedoch wieder mit `!roll` im `#multiplayer` ermittelt werden, welches Team zuerst anfängt.
-   1. Die Kapitäne können frei nach Belieben eine Beatmap aus den einzelnen Gruppen auswählen.
-   2. Im Falle eines Unentschiedens, muss die Tiebreaker Map gespielt werden.
-5. Die Ergebnisse werden im osu! Wiki und per Statistiken auf einer Webseite veröffentlicht.
+2. Der Raum wird verschlossen. Das Passwort und die Einladung zum Raum werden beiden Kaptiänen so früh wie möglich gesendet.
+3. Raumeinstellungen sind osu!, Team-Vs., Win Condition: 'Score'. Raumname muss "OWC 2014: TeamBlue vs TeamRed" lauten.
+4. Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
+5. Den Spielern steht es frei 2 Beatmaps für die Aufwärmphase zu spielen. Beatmaps mit fragwürdigem Inhalt sind verboten.
+6. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
+7. Beide Vetos müssen benutzt werden.
+8. Beide Kapitäne müssen `!roll` benutzen, um zu ermitteln, wer mit dem Bannen der beiden Beatmaps beginnen soll. Der andere Kapitän darf nicht dieselben Beatmaps bannen und muss daher andere Beatmaps auswählen, um Überlappungen zu vermeiden.
+9. Die Beatmaps werden abwechselnd von den beiden Kapitänen ausgewählt. Zuvor muss jedoch wieder mit `!roll` im `#multiplayer` ermittelt werden, welches Team zuerst anfängt.
+10. Die Kapitäne können frei nach Belieben eine Beatmap aus den einzelnen Gruppen auswählen.
+11. Im Falle eines Unentschiedens, muss die Tiebreaker Map gespielt werden.
+12. Die Ergebnisse werden im osu!-Wiki und per Statistiken auf einer Webseite veröffentlicht.
 
 ### Mappoolanweisungen
 
 1. Es wird ein Mappool für die Gruppenphase, ein Mappool für das Achtelfinale, ein Mappool für das Viertelfinale, ein Mappool für Halbfinale und ein Mappool für das Finale geben.
 2. Jeder Mappool besteht aus 5 Gruppen: NoMod, HardRock, Hidden, DoubleTime und FreeMod
```

Comment:
```suggestion
2. Jeder Mappool besteht aus 5 Stapeln: NoMod, HardRock, Hidden, DoubleTime und FreeMod
```
You've used this in the other article. Both is fine to me, jsut something that I wanted to mention.

Reply:
Was mainly just following the terms already used in this translation anyway. For specific mod pools there isn't really a proper term afaik, but lets go with this to stay consistent